EasyPost/Shipment.cs

```diff
@@ -118,6 +119,11 @@ public async Task Buy(string rateId, string? insuranceValue = null, bool withCar
                 body["insurance"] = insuranceValue;
             }
 
+            if (endShipperId != null)
+            {
+                body["end_shipper_id"] = endShipperId;
+            }
```

Comment:
I'd prefer if we simply exclude test coverage of this if statement here than adding another skipped test since we won't be testing this. See PHP for an example. It's not ideal, but it's the next best thing to call out "we meant not to test this"

Reply:
Any news on skipping the test coverage here? Since we are already in these weeds?

Reply:
I have excluded the `Buy` functions from code coverage. Unfortunately, there's not a way to exclude just a parameter from code coverage, I have to mark the whole function as excluded

Reply:
hmm... we don't want to exclude the whole function because this is arguably the most important function in the whole lib, we need to know it's getting tested and ignoring it outright for ~2 lines makes us worse off. Let's go ahead and take the hit for the uncovered lines, keep the coverage for it, and we'll circle back.

Reply:
Coverage of that function is at 50% of potential branches
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/17054780/190486330-dcfff391-3418-4664-8189-67dc673d86e3.png">